getting-started/loading-sample-data.md

```diff
@@ -1,2 +1,18 @@
 # Tutorial and Sample Data
 
+This section contains sample dataset that you can load into Hydra.
+
+## GitHub Events Dataset
+
+We prepared a dataset from the [GH Archive](https://www.gharchive.org/) that contains 9MB of events on `2015-01-01-15` that have 4000 rows.
+You can download it [here](../../.gitbook/assets/github_events.sql) and load it into Hydra:
```

Comment:
Hmm, dont love it. Is there anywhere else we can stick this file?

Reply:
We can move to another GitHub repo

Reply:
Could also gzip it, probably compresses by 90%

Reply:
I moved it to S3: https://github.com/HydrasDB/docs/pull/19/commits/a71e2ec4a4b4c2f4a8502fac1bd56ec86a8e425c